README.md

```diff
@@ -38,114 +46,106 @@ The league establishes a map pool for each season. The map pool for Season 1 of
+If a match is being casted, the caster(s) should be added to the lobby and set to "referee" status. 
 
-### Standings
-      
-Division standings are ordered by points. Teams earn points only in their weekly division match. Wins will reward +3 points while losses will not reward any points.
+### Substitutions
 
-### Season Rating
+Teams may sub in one non-rostered player for games, so long as that player is eligible to play in that league (see [Divisions](#divisions) and [Playing in Multiple Leagues](#playing-in-multiple-leagues-teams)). Substitutions can occur in between rounds - player who plays in any round is not required to play in any other rounds in the series. However, once drafting has begun, the roster is locked for that round.
 
-During season play, team's will acquire a seasonal rating. This can be viewed in the standing of the division. The rating represents how a team is doing relative to other teams. Higher ratings show higher performance. Chair League uses this rating to match your team against opponents of similar skill level within your division. Rating gained or loss depends on the outcome of the weekly match and the magnitude is related to the rating difference between the two teams.
+### Map Selection / Draft Order
 
-## Tournaments
+The Home team, as designated by the schedule, is responsible for selecting the map and hosting the lobby for the first match. The Away team will thus have the first pick in the draft. This then alternates each round, as such:
 
-### Divisions
+- Game 1: Home team selects map, Away team has first draft pick.
+- Game 2: Away team selects map, Home team has first draft pick.
+- Game 3: Home team selects map, Away team has first draft pick.
 
-Tournaments are separated by divisions. When a team joins a tournament, they will be joined by teams in the same division.
+The map selected must be from the [map pool](#map-pool) for the season. Each map may only be played once per series.
 
-### Seeding
-
-Tournament seeds are determined upon team rating. Tie-breakers are broken by who signed up first for the tournament.
+### Drafting
 
-### Rewards
+Use the in-game draft tool, set to tournament draft mode, for drafting.
 
-Rewards in tournaments unlock as more teams sign up. The rewards that can be earned from a tournament will differ based on the competitiveness, or division, of the tournament. More rewards will be given to Division 1 teams than Division 3 teams.
+### Results
 
-Only players who participated in a tournament are granted rewards for that tournament. Those players can [redeem their prizes in their account page](account_prizes_path).
+After the match finishes, either team can return to the match page and record the winner.
 
-## Matches
+If the match is recorded incorrectly, please contact paul@superjova.com to get it corrected.
 
-### Home Team
+## Seasons
```

Comment:
I thought of that, but also think people probably want to see the most relevant rules first, which would be "how do we play our matches?" instead of "how are season standings determined, etc.?"